README.md

```diff
@@ -1,23 +1,23 @@
-# Uninstall Spotify for Windows
+# Spotify Uninstaller for Windows
 
-The script aims to fully uninstall the desktop Spotify version.
+This script is designed to help you thoroughly remove the desktop version of Spotify from your Windows system.
 
 ## Usage
 
-1. Download the batch script: [Uninstall-Spotify.bat](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat).
-2. Double-click the downloaded `Uninstall-Spotify.bat` file to run it.
+1. **Download**: Get the batch script from [here](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat).
+2. **Run**: Double-click the downloaded `Uninstall-Spotify.bat` file to execute it.
 
-## Script Details
+## What the Script Does
 
-The script performs the following actions:
+The script executes the following steps:
 
-1. Resets permissions for the Spotify Update folder, if it exists.
-2. Uninstalls Spotify using a silent uninstallation process, if the executable is found in the AppData directory.
-3. Removes the Spotify AppData folder.
-4. Removes the Spotify LocalAppData folder.
-5. Deletes any SpotifyUninstall.exe if found.
-6. Displays a message based on whether Spotify was found and removed or not.
+1. **Permission Reset**: Resets permissions for the Spotify Update folder, if present.
+2. **Silent Uninstallation**: Uninstalls Spotify silently if its executable is located in the AppData directory.
+3. **AppData Cleanup**: Removes the Spotify AppData folder.
+4. **LocalAppData Cleanup**: Clears the Spotify LocalAppData folder.
+5. **Executable Deletion**: Deletes any SpotifyUninstall.exe file found.
+6. **Completion Message**: Provides feedback on whether Spotify was successfully located and removed.
 
 ## Disclaimer
 
-- Use this script at your own risk. Make sure you understand the actions it will perform on your system before running it. The author of this script is not responsible for any damage or loss caused by its usage.
+- **Caution**: Deploy this script at your own risk. Prior to execution, ensure you comprehend the actions it will take on your system. The author accepts no liability for any damage or loss arising from the use of this script.
```

Comment:
no additional words like caution or important etc ... are needed since we are inside the disclaimer header

also we are not deploying anything, it is just running the script, no deployment is required for this